ziti/cmd/ziti/cmd/edge/login.go

```diff
@@ -92,15 +92,25 @@ func (o *loginOptions) Run() error {
 
 	var host string
 	if len(o.Args) == 0 {
-		if defaultId := config.Identities[id]; defaultId != nil && !o.IgnoreConfig {
-			host = defaultId.Url
-			o.Printf("Using controller url: %v from identity '%v' in config file: %v\n", host, id, configFile)
-		} else {
-			if host, err = term.Prompt("Enter controller host[:port] (default localhost:1280): "); err != nil {
+		defaultHost := ""
+		if defaultId := config.Identities[id]; defaultId != nil && defaultId.Url != "" {
+			defaultHost = defaultId.Url
+			if o.DefaultConfig {
+				host = defaultId.Url
+				o.Printf("Using controller url: %v from identity '%v' in config file: %v\n", host, id, configFile)
+			}
+		}
+
+		if host == "" {
+			prompt := "Enter controller host[:port]: "
```

Comment:
Prompt asks for `host:port` but then attempts to parse the input as a URL and fails. 

I'm seeing this on windows as well:


```
09-03 16:44:36 (Grubby) ziti [update-deps ≡]> ziti edge login -i weeee
Enter controller host[:port]: localhost:1280
error: invalid controller URL: parse "https://localhost:1280\r": net/url: invalid control character in URL
09-03 16:44:44 (Grubby) ziti [update-deps ≡]> ziti edge login -i weeee
Enter controller host[:port]: https://localhost:1280
error: invalid controller URL: parse "https://localhost:1280\r": net/url: invalid control character in URL
09-03 16:44:53 (Grubby) ziti [update-deps ≡]>
```

Reply:
For `host:port` validation there is `validateHostPortString` in the edge controller.

Reply:
Added some whitespace trimming which should fix your issue

Reply:
It does thank you!